fails to create database and then times out at creating a http server
--- a/tests/functional-tests/meson.build
+++ b/tests/functional-tests/meson.build
@@ -35,7 +35,6 @@ functional_tests = [
   'collation',
   'ontology-changes',
   'ontology-rollback',
-  'cli',
   'portal',
 ]
 
